sources/packages/api/src/route-controllers/application/models/application.dto.ts

```diff
@@ -10,3 +10,12 @@ export interface ApplicationDetails {
   applicationLocationName: string;
   applicationStatus: string;
 }
+
+export class ActiveApplicationSummaryAPIOutDTO {
+  applicationNumber: string;
+  studyStartPeriod: string;
+  studyEndPeriod: string;
+  applicationId: number;
+  applicationStatus: string;
```

Comment:
Is applicationStatus not an enum ? 
<img width="589" alt="image" src="https://user-images.githubusercontent.com/72251620/163009614-453805de-171e-4173-8651-59b660ee6e5e.png">